src/dataflow/src/server.rs

```diff
@@ -470,6 +670,33 @@ where
                     lower.clone_from(&upper);
                 }
             }
+
+            for (id, source_ts_history) in self.render_state.ts_histories.borrow().iter() {
+                match source_ts_history {
+                    TimestampDataUpdate::RealTime(_) => continue,
+                    TimestampDataUpdate::BringYourOwn(history) => {
+                        let upper = history.get_upper();
```

Comment:
probably want to match the upstream interface where we take a mutable ref to an antichain instead of cloning a new one for each source